st2actionrunner/Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG ST2_VERSION
-FROM stackstorm/st2:${ST2_VERSION}
+ARG DOCKER_TAG
+FROM stackstorm/st2:${DOCKER_TAG}
```

Comment:
st2web and st2chatops (and base, though that's out of scope) rely on ST2_VERSION in their build for running scripts and packages. The others could just be a variable rename, but it would require some additional logic in the Makefile.

`RUN if [ "${ST2_VERSION#*dev}" != "${ST2_VERSION}" ]; then \`
`& apt-get install -y st2web=${ST2_VERSION}-* \`
`&& apt-get install -y st2chatops=${ST2_VERSION}-* \`

Reply:
I see now, thanks!

So let's then rename var that won't require any additional Makefile changes?
st2 services like `st2actionrunner`, `st2sensorcontainer`... and so on.

So we won't have unused `ST2_VERSION` var in the Dockerfiles.